torch/_inductor/codecache.py

```diff
@@ -551,40 +557,43 @@ def _reduce_fake_tensor(
         return (_ident, (metadata,))
 
     def _reduce_tensor(
-        self,
-        t: Tensor,
-    ) -> Tuple[Callable[[T], T], Tuple[Union[TensorMetadata, TensorMetadataAndValues]]]:
+        self, t: Tensor
+    ) -> Tuple[Callable[[T], T], Tuple[TensorMetadataAndValues]]:
         """
         Custom reducer to pickle Tensors.  If we see tensors, we know they're constants
         stored as attributes on the GraphModule.
         """
-        from .graph import GraphLowering
-
         if t.is_mkldnn:
             # TODO: These tensors don't currently pickle, so we can't cache a compiled
             # graph containing them. Just fail now. If mkldnn tensors get pickling
             # support, we can remove this.
             raise BypassFxGraphCache("mkldnn tensors unpickleable")
 
-        # If this is an inlined constant or include_non_inlined=True, then we include
-        # the metadata and the values.
+        # Very large tensors will be expensive to copy to cpu and hash. Let's at least
+        # report any slowness.
+        start = time()
+        values = t.tolist()
+        elapsed = time() - start
+        if elapsed > 1.0:
+            warnings.warn(
+                f"FX graph cache copying of a large constant took {elapsed:.1}s. "
+                "Please file an issue."
+            )
+
         metadata = extract_tensor_metadata_for_cache_key(t)
-        if GraphLowering.can_inline_constant(t) or self.include_non_inlined:
-            # Very large tensors will be expensive to copy to cpu and hash. Let's at
-            # least report any slowness.
-            start = time()
-            values = t.tolist()
-            elapsed = time() - start
-            if elapsed > 1.0:
-                warnings.warn(
-                    f"FX graph cache copying of a large constant took {elapsed:.1}s. "
-                    "Please file an issue."
-                )
+        return (_ident, (TensorMetadataAndValues(metadata, values),))
 
-            return (_ident, (TensorMetadataAndValues(metadata, values),))
+    def _reduce_parameter(
```

Comment:
The assumption here is that for hashing purposes, I can consider Parameters to be Tensors